README.md

```diff
@@ -5,12 +5,31 @@ make municipal finance information available to the public. It is made up of a c
 
 # Local development
 
-1. clone this repo
-2. install dependencies: ``pip install -r requirements.txt``
-3. create a postgresql user with password ``municipal_finance``: ``createuser municipal_finance -W``
-4. create a database: ``createdb municipal_finance -O municipal_finance``
-5. install data from somewhere :)
-6. run it: ``python manage.py runserver``
+1. Clone this repo
+2. Create a python virtual environment. ```virtualenv <env_name>```
+3. Activate virtual environment ``` source <env_name>/bin/activate```
+4. Install dependencies.
+
+   Some python packages require system packages.
+   * psycopg2 requires libpq-dev (postgresql development files).
+   * shapely requires libgeos-dev (Geometry engine development files).
```

Comment:
I suspect this is a hangover, and we can get rid of shapely and therefore libgeos-dev.